Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -49,22 +51,33 @@ - (void)dealloc
   RCTSRWebSocket *webSocket = [[RCTSRWebSocket alloc] initWithURL:URL];
   webSocket.delegate = self;
   webSocket.reactTag = socketID;
-  if (!_sockets) {
+  static dispatch_once_t once;
```

Comment:
AFAIK all instances will share this static variable.. better to initialize _sockets in the initializer